src/acpi_tables/src/aml.rs

```diff
@@ -0,0 +1,1988 @@
+// Copyright © 2019 Intel Corporation
```

Comment:
Right now if I understand correctly we need very few lines of aml language. Less than 5 maybe ? Considering this I don't know how much it's worth having `aml.rs` which has 2000 lines defining helper functions in order to work with AML. Could we just hardcode a couple of lines of AML and remove this file ? Do we expect to use much more AML code in the future ?